mountpoint-s3/src/inode.rs

```diff
@@ -254,6 +253,22 @@ impl Superblock {
         Ok(LookedUp { inode, stat })
     }
 
+    /// Create a new write handle to be used for state transition
+    pub async fn write<OC: ObjectClient>(
```

Comment:
Is there ever a case where we'd call this without immediately calling `start_writing` afterwards? If not we should just fold that transition into this constructor.

Reply:
Yeah, we probably don't have such case, already moved it.